README.md

```diff
@@ -134,7 +135,37 @@ you can bypass this by changing this line (in `Resources`):
   //public static Region tnr = new Region("TNR_LL_x", "TNR_LL_y", "TNR_UR_x", "TNR_UR_y");
   public static double targetAngle = Math.max(get("TNR_LL_x"), get("TNR_UR_x"));
   ```
-  Do this only for the beta demo, and remember to change it back afterwards. 
+  Do this only for the beta demo, and remember to change it back afterwards.
+
+#### &nbsp;&nbsp;&nbsp;**• For the final competition:**
+  You must change your code to receive
+`redBin` and `greenBin` parameters instead of `bin`. To do that:
+  1. Update the Server XML layout description file ([`layout.xml`](Server/layout.xml))
+  and the example file ([`example_data_fill.xml`](Server/example_data_fill.xml))
+  by pulling from this repo or redownloading the latest version from MyCourses.
+  If you had any custom XML layout data files, update them to include the two bins.
```

Comment:
I added this clarification, since students may have forgotten that they need to update the server files.